src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx

```diff
@@ -31,41 +31,17 @@ import apmPrometheus from 'images/apm_logo_prometheus.png'
 
 import kubernetesNewRelicKubernetesClusterExplorer from 'images/kubernetes_screenshot-full_new-relic-kubernetes-cluster-explorer.gif'
 
-New Relic is an observability platform that helps you build better software. You can bring in data from any digital source so that you can fully understand your system and how to improve it.
+New Relic is an observability platform that helps you build better software. You can bring in data from any digital source so that you can fully understand your system, analyze that data efficiently, and respond to incidents before they become problems. As complex as the capablities of New Relic are, you can get started with the platform by following a 3-step procedure.
```

Comment:
I wonder if there's a different word for 'complex' that captures complexity but spins it as like, Complex because it's So Good... I initially read this sentence as, 'It may be hard as hell to understand, but it's easy to set up!'

Reply:
How about "extensive"?